schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY,
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure to make the `id` not null and auto incremented 
```suggestion
    id INT NOT NULL IDENTITY(1,1) PRIMARY KEY,
```